### ISSUE TITLE
OpenSuse tumbleweed: getting weird display issues when combining bat + man

### ISSUE THREAD

**Issue by mrjayviper** [sharkdp/bat#1924]
setup:

- using OpenSuse tumbleweed
- using fish shell with the following option => set -x MANPAGER "sh -c 'col -bx | bat -l man -p'"

screenshot of the problem: https://i.imgur.com/VRQP6vA.png

Any ideas on how to fix?

I've played around with the monospaced font I'm using in terminal and it makes no difference.

Thanks

**Comment by keith-hall** [comment]
Does it make a difference if you use `set -x MANPAGER "sh -c 'col -bx | bat -l man -p --no-pager'"` (i.e. skip the pager) - it would be nice to isolate whether that is the problem.

Also, please post the output from `bat --diagnostic`

**Comment by sharkdp** [comment]
I'm closing this due to inactivity. Please feel free to comment in case it should be re-opened.

**Comment by Epicsteve2** [comment]
**The fix is to add `export MANROFFOPT='-c'` to your shell rc file**.

I had the same issue for openSUSE Tumbleweed on `zsh`. I had pretty much the exact same screenshot

Not gonna lie, I have no clue what that does :sweat_smile:. I just noticed how [batman.sh in eth-p/bat-extras](https://github.com/eth-p/bat-extras/blob/master/src/batman.sh) doesn't have this issue. I tried copying the same environment variables as that script and viola! It was fixed :partying_face:
